Eastern-Research-Group/csb-rebate-forms-app
# Feature/add playwright

## Related Issues:
* CSBAPP-476

## Main Changes:
* Add initial playwright setup and dashboard tests

## Steps To Test:
1. Run the tests (`npm run test` / `npm test` / `npm tst` / `npm t`) and ensure everything passes.

## TODO:
- [ ] This mostly just sets things up. Add more tests!
